frontend/scripts/game.js

```diff
@@ -1,12 +1,14 @@
 import createGame  from "./gameLogicInterface.js"
 import { drawGrid } from "./gameUI.js"
 
+
 initialiseGame();
 
 function initialiseGame() {
     let game = createGame();
 
     drawGrid();
+    document.getElementById("score").textContent=game.getScore()
```

Comment:
Nice! Since in the real game we would want to make sure that we can keep updating the score (and not just at the beginning when we load the page), can you create a new function called `updateScoreDisplay(gameScore)`, that takes the game score as a parameter and and updates the element?